libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -53,6 +53,7 @@ Implemented Papers
 - P2713R1 - Escaping improvements in ``std::format``
 - P2231R1 - Missing ``constexpr`` in ``std::optional`` and ``std::variant``
 - P0019R8 - ``std::atomic_ref``
+- P2389R2 - ``dextents`` Index Type Parameter
```

Comment:
Should we depart from the paper name to say "alias template dims for the extents of a mdspan" or such?
I don't know what is the common practice but I find the actual paper title not so descriptive.

Reply:
Thank you for the review. I will update the code once this comment has been addressed.

Reply:
@ldionne Would you mind share your thoughts on this?

Reply:
In general I would stick to the paper title, but in this case the paper title is.. not really helpful. I would agree with @dalg24 's suggestion.